src/pudl/metadata/fields.py

```diff
@@ -331,6 +331,14 @@
             "or charging rent to host cell antennas on transmission towers."
         ),
     },
+    "block": {
+        "type": "string",
+        "description": "Title of block of data.",
+    },
+    "block_count": {
+        "type": "integer",
+        "description": "Some blocks are repeated, `block_count` defines the index of the data block.",
+    },
```

Comment:
What is a data block? More context in these descriptions would be helpful. Is `block` a specific enough column name?

I think the single backticks will probably not render correctly, since that's markdown formatting, and this will be getting parse as RST in the documentation.

Reply:
This is basically a narrow tidy table with a bunch of key value pairs that contain company information. These key values are organized in connected blocks. I've changed the naming to hopefully make this structure more clear. Let me know if it makes more sense now.